components/scream/src/physics/rrtmgp/atmosphere_radiation.hpp

```diff
@@ -84,7 +87,7 @@ class RRTMGPRadiation : public AtmosphereProcess {
   int m_ngas;
   std::vector<ci_string>   m_gas_names;
   view_1d_real             m_gas_mol_weights;
-  GasConcs gas_concs;
+  std::vector<GasConcs> gas_concs;
```

Comment:
Do all of the chunks have the same size? If so, and the loop over chunks is serial, then why not just init a single `gas_concs` object with size of each chunk?

Reply:
Good point. Working on this improvement.